src/Auth0.php

```diff
@@ -152,6 +153,13 @@ class Auth0
      */
     protected $idToken;
 
+    /**
+     * Decoded version of the ID token
+     *
+     * @var array
+     */
+    protected $idTokenDecoded;
```

Comment:
Store a decoded version of the ID token (remains `null` if none was returned/stored).